pyanaconda/core/configuration/anaconda.py

```diff
@@ -353,6 +365,14 @@ def set_from_opts(self, opts):
         if opts.debug:
             self.anaconda._set_option("debug", True)
 
+        # Set "nosave flags".
+        if hasattr(opts, "save_input_kickstart"):
```

Comment:
It would be better to initialize these values to `None` and check that.

Reply:
Oh. My brain is still strongly typed, it seems.